Bottom: 451123dcd60445b2d235fa1e464309609153a385
Top:    029d8f7cb59e21125768c1ecb49cd38820671059
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-05 20:10:30 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index c5159ba..da2e3b3 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2752,7 +2752,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
